packages/contracts/scripts/smt-checker/smt/smt.ts

```diff
@@ -0,0 +1,124 @@
+import { globSync } from "glob";
+import fs from "fs";
+import path from "path";
+import readline from "readline";
+
+const searchContracts = async (directory) => {
+  const contracts = await globSync(`${directory}/**/*.{sol,t.sol}`);
+  return contracts.map((contract) => ({
+    path: contract.replace(directory, ""),
+    name: contract.replace(/^.*[\\/]/, "").replace(/\.[^.]+$/, ""),
+  }));
+};
+
+const updateFoundryToml = async () => {
+  const currentDirectory = process.cwd();
```

Comment:
Would be nice to have the current file and rename it like foundry.toml.backup

Reply:
added script to backup initial **foundry.toml** to **foundry.toml.backup** https://github.com/ubiquity/ubiquity-dollar/pull/702/commits/ab1a2b5b762b0520292b4d5d1d845c6dcf7fd590